modules/outbound-domain-event-handler/src/application/index.ts

```diff
@@ -67,6 +78,8 @@ import Config from '../shared/config';
         logger.info('\tDestroying outboundEventHandler handler...');
 
         await outboundEventHandler.destroy();
+        await bulkTransactionEntityRepo?.destroy();
+        await apiServer.stopServer();
```

Comment:
We should really change the `stopServer` to `destroy` to align with this elsewhere throughout our code. <-- not something that we have to do right now, but something worth thinking about.